src/api.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "https://the-nc-news.onrender.com",
+});
+
+const getArticles = () => {
+  return api.get("/api/articles").then((response) => {
+    return response.data.articles;
+  });
+};
+
+export { getArticles };
```

Comment:
Nice, you could put "/api" in the baseURL since all of your calls are going to include that.